src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -73,6 +73,8 @@
     <Compile Include="System\Reflection\Metadata\MetadataReaderProvider.cs" />
     <Compile Include="System\Reflection\Metadata\BlobContentId.cs" />
     <Compile Include="System\Reflection\Metadata\Signatures\PrimitiveSerializationTypeCode.cs" />
+    <Compile Include="System\Reflection\Metadata\TypeSystem\AssemblyDefinition.netstandard2.0.cs" Condition="'$(TargetGroup)' != 'netstandard1.1'" />
```

Comment:
@nguerrera are you okay with this naming and following the TargetGroup approach instead of the `#if !NETSTANDARD1.1` -- I personally like the TargetGroup - partial classes approach but this is your area so your call. 

Also I'm not sure if it should be named `AssemblyDefinition.netstandard2.0.cs` or `AssemblyDefinition.netstandard.cs` since this will be available on NS >= 2.0 not only on 2.0